src/services/repo/useRepoComponents.spec.tsx

```diff
@@ -0,0 +1,346 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { renderHook, waitFor } from '@testing-library/react'
+import { graphql } from 'msw'
+import { setupServer } from 'msw/node'
+import { MemoryRouter, Route } from 'react-router-dom'
+
+import A from 'ui/A'
+
+import { useRepoComponents } from './useRepoComponents'
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+})
+
+const wrapper = ({ children }: { children: React.ReactNode }) => (
```

Comment:
I've been told by Nick and Ajay to use `wrapper: React.FC<React.PropsWithChildren>` instead of how you've typed this here.